crates/sui-types/src/error.rs

```diff
@@ -41,6 +41,19 @@ macro_rules! exit_main {
     };
 }
 
+#[macro_export]
+macro_rules! log_if_err {
+    ($e:expr, $($arg:tt)+) => {
+        {
+            let res = $e;
+            if let Err(e) = &res {
+                tracing::info!($($arg)+, "{}", e);
```

Comment:
I think the issue ends up being that the info macro doesn't like 2 `,`s since if you add a comma at the end of the invocation of this macro, you'll end up with:
```
tracing::info!(arg,, "{}", e);
```

Reply:
ok that makes sense - can i consume an optional comma in the match rule?